index.html

```diff
@@ -9,7 +9,7 @@
 <body>
 
 	<div class="nav-bar" id="nav-bar">
-		<div class="logo"><img src="images/pigeon2.jpg" alt="logo" style="width:32px;height:32px;"></div>
+		<div class="logo"><img src="images/pigeon2.jpg" alt="logo">Messenger Pigeons</div>
```

Comment:
would you mind terribly putting the 'messenger pigeons' text in a separate div or an H3 with a class or something? right now it's completely tied to the image as a caption, which means I won't be able to style it separately.